package/base-files/files/etc/profile

```diff
@@ -41,3 +41,13 @@ in order to prevent unauthorized SSH logins.
 --------------------------------------------------
 EOF
 fi
+
+service() {
+if [ -f "/etc/init.d/$1" ]; then
+        "/etc/init.d/$1" "$2"
+        return $?
+else
+        ls "/etc/init.d"
+fi
```

Comment:
various issues here

* the implicit list is not obvious and jut a side effect of the code style. there is also no warning.
* what happens if $1 is null
* this is not an alias
* some initd commands take more than 1 option
* the return $? is duplicate as $? is already populated